_build/html/dev-manual/api/api-overview.html

```diff
@@ -71,13 +72,14 @@ <h3>Navigation</h3>
 <li>Location</li>
 </ul>
 </div></blockquote>
+<p>See</p>
 <p>Like the Archivematica API, the Storage Service API is written using TastyPie, and endpoints require authentication with a username and API key. See <a class="reference internal" href="#api-overview-auth"><span class="std std-ref">Authentication</span></a> for additional information.</p>
 </div>
 </div>
 <div class="section" id="using-the-apis">
 <h1>Using the APIs<a class="headerlink" href="#using-the-apis" title="Permalink to this headline">¶</a></h1>
 <p>All calls to the API endpoints require authentication with username and API key. These can be submitted as GET parameters (e.g. <code class="docutils literal notranslate"><span class="pre">?username=demo&amp;api_key=e6282adabed84e39ffe451f8bf6ff1a67c1fc9f2</span></code>) or in the  header (e.g. <code class="docutils literal notranslate"><span class="pre">Authorization:</span> <span class="pre">ApiKey</span> <span class="pre">demo:e6282adabed84e39ffe451f8bf6ff1a67c1fc9f2</span></code>).</p>
-<p>There are a number of API endpoints in the Archivematica Dashboard, some of which are proxies to the Storage Service. For resource and endpoint descriptions, consult the <a class="reference internal" href="api-reference.html#am-api-reference-rst"><span class="std std-ref">Reference</span></a></p>
+<p>There are a number of API endpoints in the Archivematica Dashboard, some of which are proxies to the Storage Service. For resource and endpoint descriptions, consult the <a class="reference internal" href="api-reference.html#am-api-reference-rst"><span class="std std-ref">Reference</span></a> section.</p>
 <div class="section" id="authentication">
 <span id="api-overview-auth"></span><h2>Authentication<a class="headerlink" href="#authentication" title="Permalink to this headline">¶</a></h2>
 <p>Archivematica’s Dashboard provides a simple cookie-based user authentication system using the Django authentication framework. Only authenticated users have access to the data.</p>
```

Comment:
Great idea for a conceptual topic in the **Using the APIs** section, @ross-spencer ! As I recall, you had a great discussion of this in your "API and AMClient Scripts Workshop" slide deck. Perhaps we could repurpose some of your material and develop a help topic around that discussion? Feel free to log an issue against this sub-repo so that we or you (if you'd like to do the honors!) can tackle this after delivery of phase 1 (extracting from the dev wiki and restructuring of API reference info).

Reply:
Thanks Michelle, I logged a related ticket here, would be a fun future project once these bits and pieces are in place :slightly_smiling_face: